homeworks/rostyslav.havuka_RostyslavHavuka/8-components-templates/album/src/app/show-image/show-image.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Image } from 'src/models';
+
+@Component({
+  selector: 'app-show-image',
+  templateUrl: './show-image.component.html',
+  styleUrls: ['./show-image.component.css'],
+})
+export class ShowImageComponent implements OnInit {
+  @Input() index: number;
+  @Input() image: Image;
+  @Input() currentImage: { index: number; image: Image };
+  @Output() onImageClick: EventEmitter<{
+    index: number;
+    image: Image;
+  }> = new EventEmitter<{ index: number; image: Image }>();
+
+  constructor() { };
+
+  ngOnInit(): void { };
+
+  isBlur(event: FocusEvent) {
```

Comment:
Do you use this method? I didn't find where. If you don't, remove the method, please